doc/ale-rust.txt

```diff
@@ -23,6 +23,11 @@ Integration Information
      over cargo. rls implements the Language Server Protocol for incremental
      compilation of Rust code, and can check Rust files while you type. `rls`
      requires Rust files to contained in Cargo projects.
+  3. analyzer -- If you have rust-analyzer installed, you might prefer using
```

Comment:
This seems to have already been messed up before you made your changes, but the line above this list says:
```
Note that there are three possible linters for Rust files:
```
but there are now 5 entries in the list and one of them isn't a linter, it's a fixer (rustfmt). The number should be changed to reflect the number of items in the list. These should either be split into two lists or the wording should be changed to reflect that there are both linters and fixers in here.